Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -492,116 +498,125 @@ class BocList
+    var fakeTableHead = realTableHead.cloneNode(true) as HTMLElement;
+    realTableHead.setAttribute('aria-hidden', 'true')
+    realTableHead.setAttribute('role', 'none')
+    realTableHead.querySelectorAll('*[role]').forEach(element =>
+    {
+      element.setAttribute('role', 'none');
     });
-    realTableHead.find ('*[role]').attr ({ 'role' : 'none' });
     fakeTable.append(fakeTableHead);
   
-    var fakeTableHeadWidthContainer = $('<div/>').attr ({ 'role' : 'none' });
+    var fakeTableHeadWidthContainer = document.createElement('div');
+    fakeTableHeadWidthContainer.setAttribute('role', 'none');
     fakeTableHeadWidthContainer.append(fakeTable);
   
-    var fakeTableHeadContainer = $('<div/>').attr({ 'role': 'none', 'class': 'bocListFakeTableHead' });
-    fakeTableHeadContainer.hide();
+    var fakeTableHeadContainer = document.createElement('div');
+    fakeTableHeadContainer.setAttribute('role', 'none');
+    fakeTableHeadContainer.classList.add('bocListFakeTableHead');
+    fakeTableHeadContainer.style.display = 'none';
     fakeTableHeadContainer.append (fakeTableHeadWidthContainer);
   
-    realTableHead.find('*').each(function (this: HTMLElement) { $(this).removeAttr('id').attr({ tabIndex: -1 }).attr({ tabIndex: -1 }); });
+
+    realTableHead.querySelectorAll('*').forEach(element =>
+    {
+      element.removeAttribute('id');
+      element.setAttribute('tabIndex', '-1');
+    });
   
     scrollableContainer.before(fakeTableHeadContainer);
   
     // sync checkboxes
-    var checkboxes = fakeTableHead.find("th input:checkbox");
-    checkboxes.click(function (this: HTMLElement)
+    var checkboxes = fakeTableHead.querySelectorAll("th input[type=checkbox]");
+    checkboxes.forEach(checkbox =>
     {
-      var checkName = $(this).attr('name');
-      var checkStatus = $(this).prop('checked');
-      $('input[name="' + checkName + '"]').prop('checked', checkStatus);
+      checkbox.addEventListener('click', ((event: MouseEvent) =>
+      {
+        var checkName = checkbox.getAttribute('name');
+        var checkStatus = (checkbox as HTMLInputElement).checked
+        document.querySelectorAll('input[name="' + checkName + '"]').forEach(element =>
+        {
+          (element as HTMLInputElement).checked = checkStatus
+        });
+      }) as EventListener)
     });
     // BocList_FixHeaderSize() with timeout needs to be called after setup. This is already taken care of at the call-site.
   }
 
-  private static FixHeaderSize(scrollableContainer: JQuery): void
+  private static FixHeaderSize(scrollableContainer: HTMLElement): void
   {
-    var realTable = scrollableContainer.children('table').first();
-    var realTableWidth = realTable[0].offsetWidth;
-    var previousRealTableWidth = realTable.data("bocListPreviousRealTableWidth");
+    var realTable = scrollableContainer.querySelector<HTMLTableElement>(':scope>table')!;
+    var realTableWidth = BocList.getComputedWidth(realTable);
```

Comment:
This should stay `.offsetWidth`

Reply:
No can do, `ScreenshotTest_TableContainer` fails if I do so